SwordSettingsTabHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file plugins/generic/sword/SwordSettingsTabHandler.inc.php
+ *
+ * Copyright (c) 2003-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SwordSettingsTabHandler
+ * @ingroup plugins_generic_sword
+ *
+ * @brief Responds to requests for SWORD settings page
+ */
+
+import('classes.handler.Handler');
+
+class SwordSettingsTabHandler extends Handler {
```

Comment:
Rather than using a new handler and a `LoadHandler` hook intervention for the plugin settings, I'd suggest using the plugin's `manage` function. See e.g. https://github.com/pkp/piwik.

Reply:
Piwik opens settings page in a modal.  

SWORD settings page contains a deposit points management grid with CRUD features. We talked about this few weeks ago and decided that SWORD settings page would reside in its own tab.

Reply:
Then maybe the static pages plugin is a closer example. It extends the tab set and uses LoadComponentHandler` instead of `LoadHandler` to extend the setup area.

Reply:
Okay. The static pages plugin is loading `StaticPageGridHandler`. In the case of SWORD plugin, there's also a form (containing the checkbox field). Can I still use `LoadComponentHandler` in this case?